ivadomed/config/config.json

```diff
@@ -70,9 +70,16 @@
         "depth": 3
     },
     "FiLMedUnet": {
-        "applied": false,
+        "applied": true,
         "metadata": "contrasts",
         "film_layers": [0, 1, 0, 0, 0, 0, 0, 0, 0, 0]
+    },
+        "UNet3D": {
+        "applied": true,
+        "length_3D": [128, 128, 16],
+        "stride_3D": [128, 128, 16],
+        "attention": false,
+        "n_filters": 8
```

Comment:
We will need to reverse this before merging, as used in the tutorial etc.

Reply:
Change here: 21947e4